src/Util/ZipFiles.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Util;
+
+use App\Entity\File;
+
+use Doctrine\Common\Collections\Collection;
```

Comment:
I'm not sure it it's better to use ArrayCollection, which have used more often in other code. But when I look at the source code of this library, they seem nearly identical.

Reply:
Ok, collection is the "interface", and arrayCollection is what you can instantiate.